lib/rules/value-no-vendor-prefix/README.md

```diff
@@ -41,3 +41,23 @@ a { max-width: max-content; }
 ```css
 a { background: linear-gradient(bottom, #000, #fff); }
 ```
+
+## Optional secondary options
+
+### `ignoreValues: ["-webkit-", "-moz-", "-ms-", "-o-"]`
+
+Given:
+
+```js
+["-webkit-", "-moz-"]
```

Comment:
The intention of the option is to ignore properties, not vendor prefixes. So here the user might specify:

```js
["grab", "max-content"]
```

To have `-webkit-grab` and `-moz-max-content` ignored from the rule.